indico/modules/events/abstracts/__init__.py

```diff
@@ -62,6 +63,7 @@ def _get_feature_definitions(sender, **kwargs):
 @signals.event_management.get_cloners.connect
 def _get_cloners(sender, **kwargs):
     yield AbstractSettingsCloner
+    yield ContributionFieldCloner
```

Comment:
this is not needed; it's already being registered in the contribution module (where it comes from)

Reply:
Done